gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -137,6 +137,10 @@
             <property name="illegalPkgs" value="org.apache.log4j, org.apache.logging.log4j, java.util.logging, ch.qos.logback"/>
             <message key="import.illegal" value="Use SLF4J instead of a logging framework directly."/>
         </module>
+        <module name="IllegalImport">
```

Comment:
is that because you think usage of guava optionals/suppliers is still too widespread?